src/sentry/notifications/utils/__init__.py

```diff
@@ -346,13 +346,15 @@ def get_parent_and_repeating_spans(
 
 
 def perf_to_email_html(
-    spans: Union[List[Dict[str, Union[str, float]]], None], problem: PerformanceProblem = None
+    spans: Union[List[Dict[str, Union[str, float]]], None],
+    problem: PerformanceProblem = None,
+    event: Event = None,
```

Comment:
Nice! FYI there's a [bug in Jira for this](https://getsentry.atlassian.net/browse/PERF-1952), and the fix should be applied to _all_ alert types at some point 🙏🏻